neuralmonkey/encoders/raw_rnn_encoder.py

```diff
@@ -0,0 +1,178 @@
+from typing import Any, Optional, Tuple
+
+import numpy as np
+import tensorflow as tf
+from typeguard import check_argument_types
+
+from neuralmonkey.encoders.attentive import Attentive
+from neuralmonkey.model.model_part import ModelPart, FeedDict
+from neuralmonkey.logging import log
+from neuralmonkey.nn.noisy_gru_cell import NoisyGRUCell
+from neuralmonkey.nn.ortho_gru_cell import OrthoGRUCell
+from neuralmonkey.dataset import Dataset
+
+# pylint: disable=invalid-name
+RNNCellTuple = Tuple[tf.contrib.rnn.RNNCell, tf.contrib.rnn.RNNCell]
+# pylint: enable=invalid-name
+
+
+# pylint: disable=too-many-instance-attributes
+class RawRNNEncoder(ModelPart, Attentive):
+    """A raw RNN encoder that gets input as a tensor."""
+
+    # pylint: disable=too-many-arguments,too-many-locals
+    def __init__(self,
+                 name: str,
+                 data_id: str,
+                 rnn_size: int,
+                 input_dimension: int,
+                 max_input_len: Optional[int]=None,
+                 dropout_keep_prob: float=1.0,
+                 attention_type: Optional[Any]=None,
+                 attention_fertility: int=3,
+                 use_noisy_activations: bool=False,
```

Comment:
vim, že je to kopie sentence enkoderu, ale tyhle věci tu být nemusí:
`attention_fertility`, `use_noisy_activations` a `parent_encoder`.. To tady trochu zjednoduší ten kód.